### PR TITLE
add Learn card for docs and general info

### PR DESCRIPTION
**Pull request by biancabuzea200**
https://app.clickup.com/t/2645698/DEV-6524

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-6524 Add 6th card on DOCS on LEARN](https://app.clickup.com/t/2645698/DEV-6524)

File: docs/learn/introduction.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+By integrating different LSPs in unique ways, LUKSO solves fundamental problems that we currently face when it comes to blockchains such as:
+
+- upgradeable security: UPs can be owned by an EOA or a custom contract such as a multisig. Therefore, you can have multiple devices and applications controlling your account, each of them with different permissions. You can even have social recovery, such that you grant certain permissions to your account to your friends and family
+- you can build your identity by attaching info to your UP account such as your name, a profile picture, description, etc., offering a web2 like-experience
+- senders and receivers are notified of transfers (e.g.: tokens transfers), and you can see which assets you own within your profile. You can even block transfers of certain assets
+- updatable and flexible metadata: metadata can be updated, enabling dynamic NFTs that can change certain properties over time, while other properties remain immutable
+- you can extend your account over time seamlessly, with no need to redeploy the associated smart contract
```
Comment (Hugoo) [inline docs/learn/introduction.md:24]: would be nice to add links to relevant pages here

File: docs/learn/introduction.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+LUKSO is L1 blockchain, EVM (Ethereum Virtual Machine) compatible, based on PoS (Proof of Stake), for creative economies.
+
+## How LUKSO started?
+
+The story of LUKSO started with ERC725, a standard proposed by Fabian Vogelsteller for blockchain-based identity. ERC725 allows for self-sovereign identity. Over the years we saw the negative effects of having a centralized identity with damaging leaks and unfair selling of user data and identity. An open, portable standard for identities enables decentralized reputation, governance, and more. Users can take their identity across different dApps and platforms that support this standard.
+LUKSO is the first blockchain that powers self-sovereign identities.
+
+## Why choose LUKSO?
+
+LSPs (LUKSO Standard Proposals) are the building blocks of LUKSO. They are used to to build blockchain-based applications that offer an enhanced user experience, and allow for more flexibility and interaction. LSPs are part of the more general LIPs used to discuss protocol specifications and client API improvements. To learn more about LSPs, check out the following [link](https://docs.lukso.tech/standards/introduction).
```
Comment (Hugoo) [inline docs/learn/introduction.md:14]: you can link to the LSP page


```suggestion
[LSPs (LUKSO Standard Proposals)](https://docs.lukso.tech/standards/introduction) are the building blocks of LUKSO. They are used to to build blockchain-based applications that offer an enhanced user experience, and allow for more flexibility and interaction. LSPs are part of the more general LIPs used to discuss protocol specifications and client API improvements. To learn more about LSPs, check out the following [link](https://docs.lukso.tech/standards/introduction).
```

**Review by Hugoo — COMMENTED** [review]
don't hesitate to add more links

-

let's make sure the content here is not "duplicated" with the FAQ section

File: docs/learn/introduction.md (hunk 1 above)
Comment (biancabuzea200) [reply docs/learn/introduction.md]: added relative links

**Review by Hugoo — APPROVED** [review]

**Pull request merged**